src/adam_test_data/observatory.py

```diff
@@ -0,0 +1,154 @@
+from dataclasses import dataclass
+from typing import Literal, Optional
+
+
+@dataclass
+class FieldOfView:
+    camera_model: str = Literal["footprint", "circle"]
+    footprint_path: Optional[str] = None
+    fill_factor: Optional[float] = None
+    circle_radius: Optional[float] = None
+    footprint_edge_threshold: Optional[float] = None
+
+    def __post_init__(self):
+        self.camera_model = self.camera_model
+
+        if self.camera_model == "footprint":
+            if self.footprint_path is None:
+                raise ValueError(
+                    "footprint_path is required for camera_model='footprint'"
+                )
+        elif self.camera_model == "circle":
+            if self.fill_factor is None:
+                raise ValueError("fill_factor is required for camera_model='circle'")
+            if self.circle_radius is None:
+                raise ValueError("circle_radius is required for camera_model='circle'")
+            if self.footprint_edge_threshold is not None:
+                raise ValueError(
+                    "footprint_edge_threshold is only valid for camera_model='footprint'"
+                )
+        else:
+            raise ValueError(f"Unknown camera model: {self.camera_model}")
+
+    def to_string(self) -> str:
+        string = "[FOV]\n"
+        if self.camera_model == "footprint":
+            string += f"camera_model = {self.camera_model}\nfootprint_path = {self.footprint_path}"
+            if self.footprint_edge_threshold is not None:
+                string += (
+                    f"\nfootprint_edge_threshold = {self.footprint_edge_threshold}"
+                )
+        else:
+            string += f"camera_model = {self.camera_model}\nfill_factor = {self.fill_factor}\ncircle_radius = {self.circle_radius}"
+
+        return string
+
+
+@dataclass
+class Simulation:
+    ang_fov: float
+    fov_buffer: float
+    picket: Optional[int] = 1
+    healpix_order: Optional[int] = 6
+
+    def to_string(self, observatory_code) -> str:
+        return f"""[SIMULATION]
+ar_ang_fov = {self.ang_fov}
+ar_fov_buffer = {self.fov_buffer}
+ar_picket = {self.picket}
+ar_obs_code = {observatory_code}
+ar_healpix_order = {self.healpix_order}"""
```

Comment:
I could be wrong, but I think you can get the multi-line no whitespace string this way:

```suggestion
return (
    f"[SIMULATION]\n"
    f"ar_ang_fov = {self.ang_fov}\n"
    f"ar_fov_buffer = {self.fov_buffer}\n"
    f"ar_picket = {self.picket}\n"
    f"ar_obs_code = {observatory_code}\n"
    f"ar_healpix_order = {self.healpix_order}"
)
```

Not really better, just different.